File: src/OnPeople/BackEnd/src/OnPeople.Domain/Models/Cargos/Cargo.cs
using OnPeople.Domain.Models.Departamentos;
using OnPeople.Domain.Models.Empresas;
using OnPeople.Domain.Models.Funcionarios;

namespace OnPeople.Domain.Models.Cargos
{
    public class Cargo
    {
        public int Id { get; set; }
        public string NomeCargo { get; set; }
        public Boolean Ativo { get; set; }
        public string DataCriacao { get; set; }
        public string DataEncerramento { get; set; }
        public int DepartamentoId { get; set; }
        public int EmpresaId { get; set; }
        public IEnumerable<Funcionario> Funcionarios { get; set; }
    }
}

File: src/OnPeople/FrontEnd/OnPeople-App/src/app/jobRoles/models/Cargo.ts
import { Empresa } from "src/app/companies/models";
import { Departamento } from "src/app/departments/models";
import { Funcionario } from "src/app/employees/models/Funcionario";


export interface Cargo {
  id: number;
  nomeCargo: string;
  ativo: Boolean;
  dataCriacao: string;
  dataEncerramento: string;
  departamentoId: number;
  departamento: Departamento;
  empresaId: number;
  empresa: Empresa;
  funcionarios: Funcionario[];
}
